examples/Cloudy/Cloudy_example.jl

```diff
@@ -216,42 +253,40 @@ mcmc = MCMC.MCMCObj(yt_sample, Γy, priors, new_step, u0, max_iter,
                     mcmc_alg, burnin, svdflag=true)
 MCMC.sample_posterior!(mcmc, gpobj, max_iter)
 
-posterior = MCMC.get_posterior(mcmc)      
+posterior = MCMC.get_posterior(mcmc)
 
-post_mean = mean(posterior, dims=1)
-post_cov = cov(posterior, dims=1)
+post_mean = get_mean(posterior)
+post_cov = get_cov(posterior)
 println("post_mean")
 println(post_mean)
 println("post_cov")
 println(post_cov)
-println("D util")
-println(det(inv(post_cov)))
-println(" ")
 
 # Plot the posteriors together with the priors and the true parameter values
-true_values = [log(N0_true) log(θ_true) log(k_true)]
-n_params = length(true_values)
+n_params = length(names)
```

Comment:
Just a note, you could use a getter here from the parameter distribution `get_total_dimension(posterior)` to obtain this - might be nice to showcase here

Reply:
I changed this to `length(get_name(posterior))` (What I need here is the number of parameters, `get_total_dimension(posterior)` would only return the correct number if all parameters are 1-D)